cves/kernel/CVE-2014-7825.yml

```diff
@@ -450,7 +444,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: This was a lapse. The lapse was forgetting to check for invalid syscall numbers when tracing syscalls.
```

Comment:
Consider adding what the syscall was for here as it would make it more clear to the reader what's going on.

Reply:
Will add this! Thanks